### ISSUE TITLE
improve accessibility

### ISSUE THREAD

**Issue by erwindon** [erwindon/SaltGUI#291]
**Describe the bug**
SaltGUI cannot be fully controlled using the keyboard only.

**To Reproduce**
The button `>_` (run command) is a click-only button because it cannot get the keyboard focus.

**Expected behaviour**
Every function should be reachable using the keyboard.

**Additional context**
See the Firefox "Accessibility" tab (from the developer tools) for detailed information.

**Comment by erwindon** [comment]
The same is true for almost every other screen element:
* table rows cannot have focus
* close button cannot have focus
* help button cannot have focus
* filter button cannot have focus
* menu bar items cannot have focus
* etc etc
too much work.
It might happen when we switch to a proper gui library, but that is not planned.
